systemvm/debian/opt/cloud/bin/setup/common.sh

```diff
@@ -759,6 +759,8 @@ parse_cmd_line() {
         ntpserverlist)
             export NTP_SERVER_LIST=$VALUE
             ;;
+            authorized_key)
+              export AUTHORIZED_KEYS=$VALUE
```

Comment:
minor nit: you want to add as:

```suggestion
       authorized_key)
            export AUTHORIZED_KEYS=$VALUE
            ;;
```